server/src/server.js

```diff
@@ -807,6 +807,10 @@ app.post("/api/set-title/:id/:domain", function(req, res) {
       return null;
     }
     shot.userTitle = userTitle;
+    // Here we remove potentially confidential data
+    if (shot.openGraph) {
+        shot.openGraph.title = null;
+    }
```

Comment:
Sorry that I wasn't clear before, but we should update `openGraph.title` to the new title.

Reply:
Sorry I misunderstood you. I hope the commit is okay now.